### PR TITLE
Add missing i18n in Explorer

### PR DESCRIPTION
**Pull request by nop33**

### CODE REVIEW

File: apps/explorer/locales/en-US/translation.json [hunk 1]
```diff
@@ -1,7 +1,8 @@
 {
   "ALPH balance": "ALPH balance",
   "Address group": "Address group",
-  "Address": "Address",
+  "Addresses_one": "Address",
+  "Addresses_other": "Addresses",
```
Comment (mvaivre) [inline apps/explorer/locales/en-US/translation.json:5]: Why using this syntax over "Address" / "Addresses"?

Comment (nop33) [reply apps/explorer/locales/en-US/translation.json:5]: Check the docs for i18next regarding plurals: https://www.i18next.com/translation-function/plurals

Comment (mvaivre) [reply apps/explorer/locales/en-US/translation.json:5]: Okay, thanks for the link. Let's hope translators understand this.

**Review by mvaivre — APPROVED** [review]

File: apps/explorer/locales/en-US/translation.json (hunk 1 above)
Comment (nop33) [reply apps/explorer/locales/en-US/translation.json:5]: They don't need to if they are going through Crowdin.I think it displays it accordingly. Let's see

**Pull request merged**